dtcli/scripts/dt.py

```diff
@@ -180,6 +181,14 @@ def extension_dev():
     pass
 
 
+@main.group(aliases=["utilities"], hidden=True)
+def utility():
+    """
+    Set of general purpose utilities.
```

Comment:
More like scripts that were commonly used internally, not sure how to convey the spirit of this subgroup